core-bundle/assets/controllers/operations-menu-controller.js

```diff
@@ -33,6 +40,7 @@ export default class extends Controller {
 
     disconnect () {
         document.removeEventListener('pointerdown', this.close);
+        document.removeEventListener('keydown', this.closeOnEscape);
     }
```

Comment:
We can completeley remove it if we use the data-actions in controller initializations